pkg/broker/handler/processors/deliver/processor_test.go

```diff
@@ -295,38 +324,10 @@ func TestDeliverFailure(t *testing.T) {
 			}
 
 			origin := newSampleEvent()
-
-			rctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
-			defer cancel()
-
-			go func() {
-				msg, resp, err := targetClient.Respond(rctx)
-				if err != nil && err != io.EOF {
-					t.Errorf("unexpected error from target receiving event: %v", err)
-				}
-				defer msg.Finish(nil)
-
-				// If with delay, we reply OK so that we know the error is for sure caused by timeout.
-				if tc.withRespDelay > 0 {
-					time.Sleep(tc.withRespDelay)
-					if err := resp(rctx, nil, &cehttp.Result{StatusCode: http.StatusOK}); err != nil {
-						t.Errorf("unexpected error from target responding event: %v", err)
-					}
-					return
-				}
-
-				// Due to https://github.com/cloudevents/sdk-go/issues/433
-				// it's not possible to use Receive to easily return error.
-				if err := resp(rctx, nil, &cehttp.Result{StatusCode: http.StatusInternalServerError}); err != nil {
-					t.Errorf("unexpected error from target responding event: %v", err)
-				}
-			}()
-
```

Comment:
Just curious, why we delete this part of test?

Reply:
This code was used to respond events. Now it's replaced by my own handler which is needed in order to reply specific response body.